syncd/FlexCounter.cpp

```diff
@@ -1303,11 +1303,18 @@ class CounterContext : public BaseCounterContext
             {
                 values.emplace_back(serializeStat(ctx.counter_ids[j]), std::to_string(ctx.counters[i * ctx.counter_ids.size() + j]));
             }
-            values.emplace_back(m_instanceId + "_time_stamp", std::to_string(time_stamp));
```

Comment:
this was previously done in a loop, and now it's done outside loop only once, this logic is different now is that intended?

Reply:
Yes. All items share the identical timestamp. Previously we write it in each record but it overwhelmed gNMI. Now we move it out of the counter table and use a single record representing it